feat/sample_configs

NeonJarbas opened the pull request:
add sample configs. companion to https://github.com/OpenVoiceOS/OVOS-plugin-manager/pull/71

note: "display_name" and "gender" keys are only metadata, not used by the engine itself but useful for the user to know

JarbasAl committed:
feat/sample_configs

add sample configs. companion to https://github.com/OpenVoiceOS/OVOS-plugin-manager/pull/71

note: "display_name" and "gender" keys are only metadata, not used by the engine itself but useful for the user to know

NeonDaniel reviewed (commented):
Should implement https://github.com/MycroftAI/mycroft-core/pull/3059/files since we're indexing language support anyways

JarbasAl committed:
feat/available_languages

add available_languages endpoint from https://github.com/MycroftAI/mycroft-core/pull/3059

improve lang configs metadata, add display_name

JarbasAl committed:
add "offline" metadata key
